src/app/components_ngrx/work-item-cell/work-item-cell.component.html

```diff
@@ -63,16 +63,17 @@
 
 <div *ngIf="col === 'creator'" class="user-avatar">
   <img placement="left"
-        tooltip="{{row.creator?.name}}"
-        src="{{row.creator?.avatar + '&s=23'}}"
-        onError="this.src='https://avatars0.githubusercontent.com/u/563119?v=3&s=23'" />
+    tooltip="{{row.creator?.name}}"
+    src="{{row.creator?.avatar + '&s=23'}}"
+    onError="this.src='https://avatars0.githubusercontent.com/u/563119?v=3&s=23'" />
 </div>
 
 <!-- Assignee -->
-<div *ngIf="col === 'assignees'" >
+<div class="assignees-cell"
+  *ngIf="col === 'assignees'">
   <f8-assignee
     [showFullName]="false"
     [truncateAfter]="3"
-    [assignees]="row.assignees ? row.assignees : []"
```

Comment:
@SMahil test is failing because of the space https://github.com/fabric8-ui/fabric8-planner/blob/master/src/tests/specs/workItemTableTest.spec.ts#L192
Could you please update test